services/drupal/web/modules/custom/view_mode_select_by_token/view_mode_select_by_token.install

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the View Mode Select by Token module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function view_mode_select_by_token_install() {
+  \Drupal::messenger()->addStatus(__FUNCTION__);
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function view_mode_select_by_token_uninstall() {
+  \Drupal::messenger()->addStatus(__FUNCTION__);
+}
+
+/**
+ * Implements hook_schema().
+ */
+function view_mode_select_by_token_schema() {
```

Comment:
This module doesn't store any data in a table so should not have this hook implemented.